crates/objc2-encode/src/helper.rs

```diff
@@ -285,6 +306,37 @@ impl<E: EncodingType> Helper<'_, E> {
         }
         Ok(())
     }
+
+    pub(crate) fn size(&self, level: NestingLevel) -> usize {
```

Comment:
I'm not sure it could easily since the `helper` method is from the `EncodingType` trait that `Helper` stores instances of and therefore cannot be `const`.